src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/Uid/config_disabled.yml

```
@@ -5,3 +5,8 @@ framework:
     http_method_override: false
     uid:
         enabled: false
+
+services:
+    Symfony\Bundle\FrameworkBundle\Test\ExceptionSubscriber:
+        tags:
+            - {name: kernel.event_subscriber}
```

```suggestion
            - { name: kernel.event_subscriber }
```
As we use whitespaces on the import above 🤷‍♂️